integration/src/test/java/org/mule/test/integration/exceptions/DefaultErrorHandlerLifecycleTestCase.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.integration.exceptions;
+
+import static org.mule.runtime.api.util.MuleSystemProperties.REUSE_GLOBAL_ERROR_HANDLER_PROPERTY;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.mule.functional.junit4.TestComponentBuildingDefinitionRegistryFactory;
+import org.mule.runtime.api.lifecycle.Disposable;
+import org.mule.runtime.api.lifecycle.Initialisable;
+import org.mule.runtime.api.lifecycle.Lifecycle;
+import org.mule.runtime.api.lifecycle.Startable;
+import org.mule.runtime.api.lifecycle.Stoppable;
+import org.mule.runtime.core.api.construct.FlowConstruct;
+import org.mule.tck.junit4.rule.SystemProperty;
+import org.mule.tests.api.LifecycleTrackerRegistry;
+
+import java.util.Collection;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.Rule;
+
+public class DefaultErrorHandlerLifecycleTestCase extends ErrorHandlerLifecycleTestCase {
```

Comment:
split (or parameterize) this test do it explicitely tests global or default error handlers

Reply:
same applies for the other test class changed in this PR